src/pages/ca-adjustment/form/CAadjustmentForms.js

```diff
@@ -131,8 +131,18 @@ export default function CAadjustmentForm({ labels, access, recordId, functionId
       if (res?.recordId) {
         toast.success('Record Posted Successfully')
         invalidate()
+
+        try {
```

Comment:
it's neccessary to add two try & catch? I suggeest to merge them to one since the catch of each are empty

Reply:
yes we can